.jenkins/PreCheckin.groovy

```diff
@@ -11,8 +11,8 @@ import java.nio.file.Path;
 
 hipCUBCI:
 {
-
-    def hipcub = new rocProject('hipCUB')
+    def buildName = 'PreCheckin'
```

Comment:
why do you have buildName as a def but not project name. Both should be ok to just define in rocProject.

Reply:
Thanks for catching that. It was left over from when I tried wrapping the load statement in another stage.